Tests/XRPKitTests/XRPKitTests.swift

```diff
@@ -180,10 +218,205 @@ final class XRPKitTests: XCTestCase {
+                            return
+                        }
+                        guard let ledgerEntryType = createdNode["LedgerEntryType"] as? String else {
+                            XCTFail("Failed retrieving created node ledger entry type")
+                            return
+                        }
+                        if validated.boolValue == true && ledgerEntryType == "PayChannel" {
+                            let channelID = createdNode["LedgerIndex"] as? String
+                            print(channelID)
+                            print(publicKey)
+                            XCTAssertNotNil(channelID)
+                            XCTAssertNotNil(publicKey)
+                            exp.fulfill()
+                        }
+                    }
+                    exp.fulfill()
+                }
+            }
+        }
+        waitForExpectations(timeout: 300)
+    }
+    
+    func testGetAccountChannels() {
+        let exp = expectation(description: "Testing \(#function)")
+        
+        let wallet = try! XRPWallet(seed: .testOneSeed)
+        let address = try! XRPAddress(rAddress: .testRAddress)
+        _ = XRPLedger.getAccountChannels(account: wallet.address, address: address.rAddress).map { (dict) in
+            print(dict)
+            exp.fulfill()
+        }
+        waitForExpectations(timeout: 5)
+    }
+    
+    func testAuthorizeChannelClaim() {
+        let exp = expectation(description: "Testing \(#function)")
+        
+        let amount = try! XRPAmount(drops: .testAmount)
+        let wallet = try! XRPWallet(seed: .testOneSeed)
+        let destWallet = try! XRPWallet(seed: .testTwoSeed)
+        _ = XRPLedger.authorizeChannelClaim(
+            channel: .testChannelID,
+            secret: wallet.seed,
+            amount: amount.drops
+        ).map { (dict) in
+            print(dict)
+            XCTAssertEqual(dict["status"] as! String, "success")
+            XCTAssertEqual(
+                dict["signature"] as! String, .successfulChannelSignature
+            )
+            exp.fulfill()
+        }
+        waitForExpectations(timeout: 5)
+    }
+    
+    func testVerifyChannelClaim() {
+        let exp = expectation(description: "Testing \(#function)")
+        
+        let amount = try! XRPAmount(drops: .testAmount)
+        _ = XRPLedger.verifyChannelClaim(
+            channel: .testChannelID,
+            signature: .successfulChannelSignature,
+            amount: amount.drops,
+            publicKey: .testChannelPublicKey
+        ).map { (dict) in
+            print(dict)
+            XCTAssertEqual(dict["status"] as! String, "success")
+            XCTAssertEqual((dict["signature_verified"] as! Int).boolValue, true)
+            exp.fulfill()
+        }
+        waitForExpectations(timeout: 5)
+    }
+    
+    func parseChannelsFromID(channels: [NSDictionary], channelID: String) -> [NSDictionary] {
```

Comment:
@MitchLang009 These can be written better and should be in the app, not just in tests...